### DISCUSSION TITLE
How to edit a role to a collaborator?

### DISCUSSION THREAD

**Discussion by veronique16** [community/community#23691] [New to GitHub]
<p>Hello. I needed to add a collaborator to my repository. I added one in “manage access”, but I didn’t get a window where I could choose a role for them. After I searched for a collaborator in the list and clicked “Select a collaborator above”, it disappered and now the collaborator is in the list. But how to edit a role for the collaborator?<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/f/7/f70ebd061a57a0a5b14a49410c928e239d9ef80b.png" data-download-href="/uploads/short-url/zfzvKdgIgUaDqoOfkDrbokjdnN9.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/73400435/181078772-1f55e379-5e06-4edf-ae89-39628b527f1b.png" alt="image" data-base62-sha1="zfzvKdgIgUaDqoOfkDrbokjdnN9" width="517" height="143" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/f/7/f70ebd061a57a0a5b14a49410c928e239d9ef80b_2_517x143.png, https://aws1.discourse-cdn.com/github/optimized/3X/f/7/f70ebd061a57a0a5b14a49410c928e239d9ef80b_2_775x214.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/f/7/f70ebd061a57a0a5b14a49410c928e239d9ef80b_2_1034x286.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/f/7/f70ebd061a57a0a5b14a49410c928e239d9ef80b_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1373×381 16.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by byrneh** [comment, marked as answer]
<p><a class="mention" href="/veronique16">@veronique16</a>, you did not clarify in your question who if the the repository was owned by a user account</p>
<p>See Collaborator access on a repository owned by a user account on the link below</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">
      <a href="https://docs.github.com/en/free-pro-team@latest/github/setting-up-and-managing-your-github-user-account/permission-levels-for-a-user-account-repository#collaborator-access-on-a-repository-owned-by-a-user-account" target="_blank" rel="noopener nofollow ugc">docs.github.com</a>
  </header>
  <article class="onebox-body">
    <img src="" class="thumbnail" width="16" height="16">

<h3><a href="https://docs.github.com/en/free-pro-team@latest/github/setting-up-and-managing-your-github-user-account/permission-levels-for-a-user-account-repository#collaborator-access-on-a-repository-owned-by-a-user-account" target="_blank" rel="noopener nofollow ugc">Permission levels for a user account repository - GitHub Docs</a></h3>



  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>
